Cargo.toml

```diff
@@ -139,3 +139,6 @@ itertools = { version = "0.12", default-features = false }
 insta = "1.8"
 tempfile = "3.4"
 tikv-jemallocator = "0.5"
+
+[patch.crates-io]
+fuel-vm = { path = "../fuel-vm/fuel-vm" }
```

Comment:
?

Reply:
The PR requires https://github.com/FuelLabs/fuel-vm/pull/796 as well=)